src/components/video.tsx

```diff
@@ -104,6 +106,22 @@ export const Video = (props: VideoProps) => {
     scrollCurrentVideo();
   }, [currentIndex]);
 
+  useEffect(() => {
+    // listening for shortcut keys
+    // Shift + n for next video and Shift + p for the prev video
+    window.addEventListener("keydown", event => {
+      if (event.key === "Shift")
+        setKeyPressed("shift")
+      else {
+        if(keyPressed === "shift" && (event.key === "N" || event.key === "n")) 
```

Comment:
Nitpick: Rather than handling individual cases, we can listen to the [event code](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code) - `KeyN` which maps to the physical key itself.

https://keycode.info

Reply:
This is cool!
Changed it to `event.code`